app/core/src/components/authentication/EmailLoginForm.js

```diff
@@ -69,14 +69,21 @@ export default class EmailLoginForm extends React.Component<Props, State> {
         placeholder={<Translation id="core.authentication.password" />}
         secureTextEntry={true}
       />
-      {this.state.loading ? (
-        <TextButton title={<Translation id="core.authentication.login" />} />
-      ) : (
-        <TextButton
-          onPress={this.handleFormSubmit}
-          title={<Translation id="core.authentication.logging_in" />}
-        />
-      )}
+
+      <TextButton
+        onPress={this.handleFormSubmit}
+        disabled={this.state.loading}
+        title={
+          <Translation
+            id={
+              this.state.loading
+                ? 'core.authentication.logging_in'
+                : 'core.authentication.login' // TODO: Does not seem to toggle when loading changes
```

Comment:
I noticed - something is broken. It's not important now but do you have any idea what's wrong?